src/components/cursor.js

```diff
@@ -50,8 +50,13 @@ module.exports.Component = registerComponent('cursor', {
 
   emit: function (evt) {
     var intersectedEl = this.intersectedEl;
-    this.el.emit(evt, { target: this.intersectedEl });
-    if (intersectedEl) { intersectedEl.emit(evt); }
+    var dispatchEvent = Object.assign({}, this.intersectionEvent, {
```

Comment:
we have an npm package that polyfills `Object.assign` where it's not natively supported. to use it, call `utils.extend`.